sdk/ai/azure-ai-inference/README.md

```diff
@@ -0,0 +1,231 @@
+# Azure Model client library for Java
```

Comment:
Should we call this "Azure Inference client library..."?

Reply:
[Updated](https://github.com/Azure/azure-sdk-for-java/pull/41482/commits/e16676f12ecb82d477715c3343df29291a46e831)